setup-documentation.sh

```diff
@@ -207,7 +207,7 @@ done
 rm -f $INSTALL_PATH/*
 
 # create Sailfish documentation package
-for NAME in $(ls $OPT_DOCDIR/sail*.qch $OPT_DOCDIR/lipstick*.qch $OPT_DOCDIR/nemo*.qch); do
+for NAME in $(ls $OPT_DOCDIR/sdk-* $OPT_DOCDIR/*sailfish*.qch $OPT_DOCDIR/lipstick*.qch $OPT_DOCDIR/nemo*.qch); do
```

Comment:
Gets longer and longer, consider something like this:

`$OPT_DOCDIR/{sdk-,*sailfish,lipstick,nemo}*.qch`

Reply:
Nice idea, updated